src/modules/combinedFacets/__tests__/CombinedFacets.test.js

```diff
@@ -142,3 +144,61 @@ test('Test Combined Facet category click', async () => {
         expect(getByText("Vasque Breeze All-Terrain GTX Womens Hiking Boots - Gargyle")).toBeInTheDocument();
     });
 });
+
+test('Test selected facet click on range Facet', async () => {
+    const { getByText, container } = render(
+        <>
+            <UnbxdSearchWrapper
+                siteKey="wildearthclone-neto-com-au808941566310465"
+                apiKey="e6959ae0b643d51b565dc3e01bf41ec1"
+            >   
+                <FacetActions showApplyFilter={false} showClearFilter={false}/>
+                <SelectedFacets />
+                <CombinedFacets />
+                <Products
+                    attributesMap={attributesMap}
+                />
+                <div>
+                    <SearchBox defaultSearch="shoes" />
+                </div>
+            </UnbxdSearchWrapper>
+        </>
+    );
+
+    await waitFor(async () => {
+        expect(getByText('$ 200 - $ 300 - 60')).toBeInTheDocument();
+        fireEvent.click(getByText("$ 200 - $ 300 - 60"));
+    });
+    await waitFor(() => {
+        expect(container.getElementsByClassName('UNX-selectedFacets__container').length).toBe(1)
+    });
+});
+
+test('Test selected facet click on text Facet', async () => {
+    const { getByText , container } = render(
+        <>
+            <UnbxdSearchWrapper
+                siteKey="wildearthclone-neto-com-au808941566310465"
+                apiKey="e6959ae0b643d51b565dc3e01bf41ec1"
+            >   
+                <SelectedFacets />
+                <CombinedFacets />
+                <Products
+                    attributesMap={attributesMap}
+                />
+                <div>
+                    <SearchBox defaultSearch="shoes" />
+                </div>
+            </UnbxdSearchWrapper>
+        </>
+    );
+
+    await waitFor(async () => {
+        expect(getByText('Scarpa - 18')).toBeInTheDocument();
+        fireEvent.click(getByText("Scarpa - 18"));
+    });
+    await waitFor(() => {
+        expect(container.getElementsByClassName('UNX-selectedFacets__container').length).toBe(1)
+        fireEvent.click(getByText("Clear"));
```

Comment:
Also check after click on clear btn, UNX-selectedFacets__container length must be 0.

Reply:
done